grants/grants.md

```diff
@@ -85,7 +85,7 @@ The Web3 foundation is interested in providing grants in the different areas of
 The grant application can be seen as a 5-step process. Below is an outline of these 5 steps.
   
 1. **Communication**
-   * See the list all of all programmes, bounties and RfPs (in a previous section).
+   * See the list of all programmes, bounties and RfPs (in a previous section).
```

Comment:
Ok, so the previous section is "Areas of Interest". Perhaps we can fix that in this PR too.

https://github.com/w3f/Web3-collaboration/blob/master/grants/grants.md#areas-of-interest